File: Range-Addition-II.cpp
class Solution {
public:
    int maxCount(int m, int n, vector<vector<int>>& x) {
        int a = m;
        int b = n;
        int y = x.size();
        
        for (int i = 0; i < y; i++) {
            if (x[i][0] < a) {
                a = x[i][0];
            }
        }

        for (int i = 0; i < y; i++) {
            if (x[i][1] < b) {
                b = x[i][1];
            }
        }

        return a * b;
    }
};

File: Range-Addition-II.py
class Solution(object):
    def maxCount(self, m, n, x):
        \\\
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        \\\
        if not x:
            return m * n

        a = m
        b = n

        for op in x:
            a = min(a, op[0])
            b = min(b, op[1])

        return a * b
